actions/snowflake-cortex-analyst/package.yaml

```diff
@@ -21,6 +21,14 @@ dependencies:
   - snowflake=1.1.0
   - pyjwt=2.10.1
 
+external-endpoints:
+  - name: "Snowflake API"
+    description: "Access Snowflake API."
+    additional-info-link: "https://docs.snowflake.com/en/guides"
+    rules:
+      - host: "*.snowflakecomputing.com"
```

Comment:
These are a bit tricky.. if executed inside SPCS that do not need egress rules, because they are already inside... not sure
I think we need to ask about these Cortex things.